pwndbg/regs.py

```diff
@@ -97,12 +97,23 @@ def __iter__(self):
         for r in self.all:
             yield r
 
+arm_cpsr_flags = { 'N': 31, 'Z': 30, 'C': 29, 'V': 28, 'Q': 27, 'J': 24, 'E': 9, 'A': 8, 'I': 7, 'F': 6, 'T': 5 }
+arm_xpsr_flags = { 'N': 31, 'Z': 30, 'C': 29, 'V': 28, 'Q': 27, 'T': 24, }
```

Comment:
The xpsr flags listed here ignore things like IRQn and IT fields which would only clutter context output.

Reply:
Good call. Showing too many flags can be too much. 
Thanks for adding the arm_cpsr_flags here, I had no idea that we didn't already have that.

Do you think it's useful to show 8,7,6 from cpsr? If showing them is regularly useful, it makes sense to have it, but otherwise it might just cause some clutter.

Reply:
I can't think of a nice way to exclude them from ```context``` while showing them in ```cpsr```, so I'd leave them. In particular the interrupt disable flag sounds useful to me if you're breaking inside some interrupt handler on embedded. Maybe it'd make sense to use an OrderedDict for flags and just display them last?

Reply:
Ok, I made them all OrderedDicts and that seems to work fine. This way AIF are at the back, and also J (24) and T (5) are grouped together on non-cortex-m.